renderer/components/ConsoleOutput.tsx

```diff
@@ -1,41 +1,54 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Panel } from 'react-bootstrap';
 
-class ConsoleOutput extends React.Component {
-  constructor(props) {
+interface StateProps {
+  toggleConsole: Function;
+  disableScroll: boolean;
+}
+
+interface OwnProps {
+  height: number;
+  output: [];
+  show: boolean;
+}
+
+type Props = StateProps & OwnProps;
+
+export class ConsoleOutput extends React.Component<Props, any> {
+  constructor(props: Props) {
     super(props);
-    this.scrollToBottom = this.scrollToBottom.bind(this);
   }
 
-  componentDidMount() {
+  componentDidMount = () => {
     this.scrollToBottom();
   }
 
-  componentWillReceiveProps(nextProps) {
-    if (this.props.output.size === 0 && nextProps.output.size > 0 && !this.props.show) {
+  componentWillReceiveProps = (nextProps: Props) => {
+    if (this.props.output.length === 0 && nextProps.output.length > 0 && !this.props.show) {
```

Comment:
Previously it was `output.size` (output is an array). but I changed it to .length (I am assuming it was getting the size of the array output?)

Reply:
Oh yes if output was an array, then we should definitely use `.length` here instead of size.